mmv1/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	golang.org/x/exp v0.0.0-20240222234643-814bf88cf225
-	gopkg.in/yaml.v3 v3.0.1
```

Comment:
From what you can tell, do we lose anything by downgrading?

Reply:
I don't see diffs after downgrading.